### PR TITLE
add about page

### PR DESCRIPTION
**Pull request by Anastasia-Tumilovich**
I didn't know what to write, so I wrote what you read:)

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>My first page</h1>
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt nisi <del>quibusdam</del> ratione consequuntur, porro tempora veniam inventore sint, natus fuga, obcaecati voluptatem quam placeat enim dicta <ins>tempore</ins> velit tenetur? Odit.</p>
+    <h2>My first page</h2>
+    <p>Lorem, ipsum dolor sit amet consectetur <sub>adipisicing</sub> elit. Totam dolor officia similique quas excepturi alias tempore adipisci fugit, quod iusto dignissimos officiis. Nostrum quod et expedita <sup>iusto</sup> aliquam! Similique, natus!</p>
+    <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <h3>My first page</h3>
+    <p><mark>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Beatae ipsa suscipit ad, nam sint alias quibusdam a quam quo adipisci quis vitae necessitatibus praesentium nesciunt assumenda, repudiandae id iure! Repellendus?</mark></p>
+    <p>
+        <blockquote><em>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Magnam, adipisci praesentium quisquam in aliquam eligendi harum veniam culpa, numquam asperiores perspiciatis nam illum veritatis sapiente atque pariatur ipsam cupiditate iure.</em><br>&copy;
+        <strong>Anastasia</strong></blockquote>
+    </p>
+    <hr>
+    <p>
+        <a href="mailto:tumilovichanastasia@gmail.com" target="_blank">My mail</a><br>
```
Comment (ghost) [inline about/index.html]: Each new tag writes by a new line.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <a href="https://github.com/Anastasia-Tumilovich" target="_blank">My github</a><br>
```
Comment (ghost) [inline about/index.html]: See the previous comment.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Моя первая страница</h1>
+    <p><a href="about/index.html">Об авторе</a></p>
```
Comment (ghost) [inline index.html]: Each new tag writes by a new line. Seems the path should be like that.
```suggestion
    <p><a href="./about/index.html">Об авторе</a></p>
```

File: about/index.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ghost) [inline about/index.html]: Please change the title.

File: about/index.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+</head>
+<body>
+    <h1>My first page</h1>
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt nisi <del>quibusdam</del> ratione consequuntur, porro tempora veniam inventore sint, natus fuga, obcaecati voluptatem quam placeat enim dicta <ins>tempore</ins> velit tenetur? Odit.</p>
```
Comment (ghost) [inline about/index.html]: If the line became very long, go to the next line.

File: about/index.html (hunk 1 above)
Comment (Anastasia-Tumilovich) [reply about/index.html]: closed all tags

File: about/index.html (hunk 2 above)
Comment (Anastasia-Tumilovich) [reply about/index.html]: moved tags to a new line

File: about/index.html (hunk 3 above)
Comment (Anastasia-Tumilovich) [reply about/index.html]: moved tags to a new line

File: index.html (hunk 4 above)
Comment (Anastasia-Tumilovich) [reply index.html]: changed the path

File: about/index.html (hunk 5 above)
Comment (Anastasia-Tumilovich) [reply about/index.html]: changed the title

File: about/index.html (hunk 6 above)
Comment (Anastasia-Tumilovich) [reply about/index.html]: maked lines shorter

**Comment by ghost** [comment]
Good job.

**Pull request merged**